client/my-sites/pages/helpers.js

```diff
@@ -53,3 +54,5 @@ export const sortPagesHierarchically = ( pages, homepageId = 0 ) => {
 
 	return sortedPages;
 };
+
+export const recordEvent = ( event ) => recordGoogleEvent( 'Pages', event );
```

Comment:
Small detail, seems that somehow the `recordGoogleEvent` is not gone through the analytics middleware. I discover this by enabling debug from the console.log and found that the following debug messages did not show for me when interacting with `<VirtualPage/>`, but showed when interacting with `<Page/>`:

![Screen Shot 2022-10-19 at 11 23 43 AM](https://user-images.githubusercontent.com/797888/196590436-f852609e-41e5-49a2-a8d2-6b6200f0998b.png)

Reply:
The reason is I call `recordGoogleEvent` function directly instead of dispatching an action. I'll fix it later!